steely/plugins/login.py

```diff
@@ -0,0 +1,6 @@
+from steely import config
+
+COMMAND = ".login"
+
+def main(bot, author_id, thread_id, thread_type, **kwargs):
+    bot.sendMessage(config.EMAIL, config.PASSWORD, thread_id=thread_id, thread_type=thread_type)
```

Comment:
Does `sendMessage` work with varargs? You might need to concatenate the email and password into a single string first.

Reply:
Not sure, I'll look at it when I'm home, you're probably right